osu.Game.Rulesets.Taiko/Objects/Drawables/DrawableHit.cs

```diff
@@ -209,6 +209,9 @@ protected override void UpdateHitStateTransforms(ArmedState state)
 
                     this.ScaleTo(0.8f, gravity_time * 2, Easing.OutQuad);
 
+                    // TODO: This "0.015" value should probably be replaced with whatever value Stable uses 
+                    MainPiece.MoveToX(-X, 0.015, Easing.None); // Visually snap hit object to hit target
```

Comment:
There's no animation in stable. It's immediately moved to the target. Same with TnT. This should be easily visible from watching a video taken from stable, so I have to check: are you somehow seeing some animation?

Reply:
I swear in stable it feels like it's moved towards the hit target for like a frame or two

Reply:
Please use video frames rather than a feelings where possible.

Reply:
Got it